### ISSUE TITLE
Unable to install other plugins due to dependencies

### ISSUE THREAD

**Issue by timprint** [percipioglobal/craft-notifications#66]
I am trying to install Solspace's Express Forms in a Craft 3.6.11 site via composer but get this error.

```
Problem 1
    - Root composer.json requires percipioglobal/craft-notifications 1.1.3 -> satisfiable by percipioglobal/craft-notifications[1.1.3].
    - percipioglobal/craft-notifications 1.1.3 requires ramsey/uuid ^3.7 -> satisfiable by ramsey/uuid[3.7.0, ..., 3.9.3].
    - You can only install one version of a package, so only one of these can be installed: ramsey/uuid[3.7.0, ..., 3.9.3, 4.1.1].
    - solspace/craft-express-forms 1.1.0 requires ramsey/uuid ^4.1.1 -> satisfiable by ramsey/uuid[4.1.1].
    - Root composer.json requires solspace/craft-express-forms ^1.1 -> satisfiable by solspace/craft-express-forms[1.1.0].
```

Any chance of getting notifications updated to use ramsey/uuid 4.1.1 ?

**Comment by timprint** [comment]
Further to that you may be able to remove it completely.
(https://discord.com/channels/456442477667418113/575387375610626063/824990472207859742)

**Comment by khalwat** [comment]
Probably you don't need the `ramsey/uuid` dependency at all, and can just remove the package. This is a good thing in terms of reducing dependencies to avoid situations like this.

You can just leverage the built-in UUID generator that Craft CMS has built in:

https://github.com/craftcms/cms/blob/develop/src/helpers/StringHelper.php#L1874

It's been available since Craft CMS 3.0.0

**Comment by cookie10codes** [comment]
@timprint I've updated the dependencies. A new version is published https://github.com/percipioglobal/craft-notifications/releases/tag/1.1.4

If you still have difficulties, please let us know

**Comment by cookie10codes** [comment]
> Probably you don't need the `ramsey/uuid` dependency at all, and can just remove the package. This is a good thing in terms of reducing dependencies to avoid situations like this.
> 
> You can just leverage the built-in UUID generator that Craft CMS has built in:
> 
> https://github.com/craftcms/cms/blob/develop/src/helpers/StringHelper.php#L1874
> 
> It's been available since Craft CMS 3.0.0
> 
> ref: [solspace/craft-express-forms#74](https://github.com/solspace/craft-express-forms/issues/74)

@khalwat thanks for this. We've opened an issue for ourselves to update in the next revision 👌🏼

**Comment by cookie10codes** [comment]
fixed in [release 1.2.2](https://github.com/percipioglobal/craft-notifications/releases/tag/1.2.2) that went out today, sorry for the delay
